README.md

```diff
@@ -273,9 +282,150 @@ dependencies:
 
 ## Editor Integration
 
-Note to editor integrators: there is now a haskell-language-server-wrapper executable, which is installed alongside the haskell-language-server executable. When this is invoked in the project root directory, it attempts to work out the GHC version used in the project, and then launch the matching haskell-language-server executable.
+Note to editor integrators: there is a haskell-language-server-wrapper executable, which is installed alongside the haskell-language-server executable. When this is invoked in the project root directory, it attempts to work out the GHC version used in the project, and then launch the matching haskell-language-server executable.
 
-All of the editor integrations assume that you have already installed haskell-language-server (see above) and that the installation script put the haskell-language-server binary in your path (usually `~/.local/bin` or `~/.cabal/bin` on linux and macOS).
+All of the editor integrations assume that you have already installed haskell-language-server (see above) and that the installation script put the haskell-language-server and haskell-language-server-wrapper binaries in your path (usually `~/.local/bin` or `~/.cabal/bin` on linux and macOS, `%APPDATA%\local\bin` or `%APPDATA%\cabal\bin` in windows).
+
+### Using Haskell Language Server with VS Code
+
+Install from
+[the VSCode marketplace](https://marketplace.visualstudio.com/items?itemName=alanz.vscode-hie-server), or manually from the repository [vscode-hie-server](https://github.com/alanz/vscode-hie-server).
+
+Choose `haskell-language-server` in the extension setting `languageServerHaskell.hieVariant`.
+
+### Using Haskell Language Server with Sublime Text
+
+- Make sure haskell-language-server and haskell-language-server-wrapper are installed (see above) and they are in the global `$PATH`.
+- Install [LSP](https://packagecontrol.io/packages/LSP) using [Package Control](https://packagecontrol.io/)
+- From Sublime Text, press Command+Shift+P and search for Preferences: LSP Settings
+- Paste in these settings. Make sure to change the command path to your `hie`
+
+```json
+{
+"clients": {
```

Comment:
Shouldn't it be indented once here? E.g.
```js
{
  "clients": {
    "haskell-ide-engine": {
      "command": ["haskell-language-server-wrapper", "--lsp"],
      "scopes": ["source.haskell"],
      "syntaxes": ["Packages/Haskell/Haskell.sublime-syntax"],
      "languageId": "haskell",
    },
  },
}
```

Also, is it normal that there are commas at the end of each line? I think some systems support it, but just wanted to be sure

Reply:
I think the name should be changed from `haskell-ide-engine` to `haskell-language-server`

Reply:
Well, not sure if those keys are hardcoded or can be changed. We are using plugins that are intended to be usable by hie.

Reply:
I tested earlier today (installed Sublime etc. as I'm not a Sublime user), and I can confirm this works well with whatever name you choose: actually this is used by Sublime LSP plugin to help the user choose which LSP to enable, as per my screenshot:
![image](https://user-images.githubusercontent.com/1204125/82950711-15f53380-9fa6-11ea-9d8a-c33720ad3fd3.png)

Reply:
Wow thanks for trying it, will change